playbooks/roles/ocp-compliance/README.md

```diff
@@ -0,0 +1,56 @@
+ocp-compliance
+=========
+
+The Compliance Operator lets OpenShift Container Platform administrators describe the required compliance state of a cluster and provides them with an overview of gaps and ways to remediate them. The Compliance Operator assesses compliance of both the Kubernetes API resources of OpenShift Container Platform, as well as the nodes running the cluster. The Compliance Operator uses OpenSCAP, a NIST-certified tool, to scan and enforce security policies provided by the content.
+
+This playbook has two modules:
+Installation of the Compliance Operator and oc-compliance plugin.
+Running the end to end tests for Compliance operator with CIS profiles and oc-compliance plugin.
+
+Requirements
+------------
+
+Access to the cluster as a user with the cluster-admin role.
+The cluster is in a known good state, without any errors.
+
+Role Variables
+--------------
+
+Role Variables
+--------------
+| Variable                       | Required | Default     | Comments                                       |
+|--------------------------------|----------|-------------|------------------------------------------------|
+| compliance_enabled             | yes      |    false    | Set it to true to run this playbook            |
```

Comment:
Same comment as previous PR... IMO cannot be Required and default needs to be defined.

Reply:
I have made the changes